### PR TITLE
Italian translation updates

### PR DESCRIPTION
**Pull request by alef**
#367 Some extra files had executable bit set, I had to flag them in git for windows in order to go on. You may want to revert them.

### CODE REVIEW

File: addons/hot/stringtable.xml [hunk 1]
```diff
@@ -5,88 +5,109 @@
...
+            <Italian>3x HOT 1 [ACE]</Italian>
         </Key>
         <Key ID="STR_ACE_HOT_hot1_4">
             <English>4x HOT 1 [ACE]</English>
             <Polish>4x HOT 1 [ACE]</Polish>
+            <Italian>4x HOT 1 [ACE]</Italian>
         </Key>
         <Key ID="STR_ACE_HOT_hot2_1">
             <English>1x HOT 2 [ACE]</English>
             <Polish>3x HOT 2 [ACE]</Polish>
```
Comment (dedmen) [inline addons/hot/stringtable.xml]: Unrelated to PR: Why is this 3x??

File: addons/vehicles/stringtable.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dedmen) [inline addons/vehicles/stringtable.xml]: BOM Change? Didn't we already fix the BOM's of all stringtables?

File: addons/grenades/stringtable.xml [hunk 3]
```diff
@@ -376,26 +376,32 @@
         <Key ID="STR_ACE_Grenades_parachute_flare_description">
             <English>Type: Star Parachute Flare</English>
             <Polish>Typ: Gwiezdna Flara Spadochronowa</Polish>
+            <Italian>Tipo: Granata a paracadute con bengala</Italian>
         </Key>
         <Key ID="STR_ACE_Grenades_40mm_flare_white">
             <English>M583 Illumination Flare (White)</English>
             <Polish>M583 Flara oświetlająca (Biała)</Polish>
+            <Italian>M583 Granata con bengala (bianca)</Italian>
```
Comment (dedmen) [inline addons/grenades/stringtable.xml]: Is it correct that `bianaca` is lowercase? The others are all first letter uppercase.

**Comment by dedmen** [comment]
What's that change with the images from docs?

**Comment by alef** [comment]
> What's that change with the images from docs?

Issues with permission in git for windows, for some reason it sets them to executables.

File: addons/vehicles/stringtable.xml (hunk 2 above)
Comment (alef) [reply addons/vehicles/stringtable.xml]: I save stringtables with tabler without checking later for BOM changes.

**Comment by PabstMirror** [comment]
Fixed chmod stuff, I had to do a force push, so be aware if adding more commits to the branch (you will probably want to delete the branch and make a new one)
I'm not sure why it happened, but you might want to run `git config core.filemode false`

**Pull request merged**